### PR TITLE
[newrelic-infrastructure-v3] deprecate chart

### PR DESCRIPTION
**Pull request by paologallinaharbur**
#### Is this a new chart
no
#### What this PR does / why we need it:
Adding a deprecation note to nri-kubernetes sinice we are moving it to a different repo

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: charts/newrelic-infrastructure-v3/README.md [hunk 1]
```diff
@@ -1,102 +1,6 @@
-[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
+DEPRECATED!! This version of the chart has been deprecated.
+Please use `nri-bundle` or directly the chart hosted in newrelic/nri-kubernetes
```
Comment (paologallinaharbur) [inline charts/newrelic-infrastructure-v3/README.md:2]: change wording

**Comment by paologallinaharbur** [comment]
Following approach of https://github.com/helm/charts for deprecating charts

File: charts/newrelic-infrastructure-v3/Chart.yaml [hunk 2]
```diff
@@ -1,27 +1,19 @@
...
+
 apiVersion: v2
 name: newrelic-infrastructure-v3
-description: A Helm chart to deploy the New Relic Kubernetes monitoring solution
-version: 3.1.0
+description: "DEPRECATED: Moved to https://newrelic.github.io/nri-kubernetes"
+version: 3.1.1
 appVersion: 3.1.0
 kubeVersion: ">=1.16.0-0"
+deprecated: true
```
Comment (roobre) [inline charts/newrelic-infrastructure-v3/Chart.yaml:11]: TIL

File: charts/newrelic-infrastructure-v3/README.md [hunk 3]
```diff
@@ -1,102 +1,6 @@
-[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
+DEPRECATED!! This version of the chart has been deprecated.
```
Comment (roobre) [inline charts/newrelic-infrastructure-v3/README.md:1]: I would make this a Lvl1 header and add some scary ⚠️ emojos

File: charts/newrelic-infrastructure-v3/README.md (hunk 1 above)
Comment (roobre) [reply charts/newrelic-infrastructure-v3/README.md:2]: Should we add a links here?

**Review by roobre — COMMENTED** [review]
Also something that comes to my mind is that artifacthub.io has some fancy annotations or sugar for flagging deprecation/replacements.

File: charts/newrelic-infrastructure-v3/README.md (hunk 1 above)
Comment (kang-makes) [reply charts/newrelic-infrastructure-v3/README.md:2]:
```suggestion
Please use `nri-bundle` or directly the chart hosted in [newrelic/nri-kubernetes
](https://github.com/newrelic/nri-kubernetes/)
```

**Review by kang-makes — APPROVED** [review]
If I were you I would spend a couple more minutes nit-picking but LGTM.

**Pull request closed without merging**